src/service/mercury/index.test.ts

```diff
@@ -259,52 +265,4 @@ describe("Mercury Service", () => {
     }
     expect(mockMercuryClient.accountSubscription).toHaveBeenCalled();
   });
-
-  it("will correctly handle missing token balance subscriptions", async () => {
```

Comment:
this test is no longer relevant because current data APIs don't need to check for the existence of a sub.